### DISCUSSION TITLE
More Updates Needed

### DISCUSSION THREAD

**Discussion by shamxeed** [flyerhq/react-native-chat-ui#71] [Q&A]
Can you please look back at this Lib as it has been almost a year since a new version was published?
And as the description of this Lib goes... "Actively maintained...."

**Comment by demchenkoalex** [comment]
As I said under some issue, I have 2 libs (Flutter and React Native) and I barely able to maintain Flutter lib at this point (it is much more widely used, so no ❤️ for React Native), so no updates are planned here, at least for now. If I will come back here I will definitely wipe out everything and make it from scratch, cause I learned a lot while maintaining the other one.
